### PR TITLE
Update Libresprite version

### PR DESCRIPTION
**Pull request by echometerain**

### CODE REVIEW

File: packages/libresprite-git/libresprite-git.pacscript [hunk 1]
```diff
@@ -1,7 +1,7 @@
 maintainer="echometerain <echometer@disroot.org>"
 
 name=libresprite-git
-version=1.2.1-Beta
+version=1.0-beta.1
```
Comment (wizard-28) [inline packages/libresprite-git/libresprite-git.pacscript:4]: Use `pkgver` function.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/libresprite-git/libresprite-git.pacscript (hunk 1 above)
Comment (echometerain) [reply packages/libresprite-git/libresprite-git.pacscript:4]: good idea

**Review by wizard-28 — CHANGES_REQUESTED** [review]
**FROM THE AUR**
```bash
	install -Dm644 "$srcdir/libresprite.desktop" \
		"$pkgdir/usr/share/applications/libresprite.desktop"
	install -Dm644 "$pkgdir/usr/share/libresprite/data/icons/ase64.png" \
		"$pkgdir/usr/share/pixmaps/libresprite.png"
```

Convert these lines to Pacstall compatible ones, and add this functionality in `install()`

File: packages/libresprite-git/libresprite-git.pacscript (hunk 1 above)
Comment (wizard-28) [reply packages/libresprite-git/libresprite-git.pacscript]: Fixed in https://github.com/pacstall/pacstall-programs/pull/330/commits/d32a36add269b450478910f11524f04f32eefe50

**Pull request closed without merging**